src/app/community/community.component.ts

```diff
@@ -33,6 +33,7 @@ export class CommunityComponent implements OnInit, OnDestroy {
   showNewsButton = true;
   deleteMode = false;
   onDestroy$ = new Subject<void>();
+  isCommunityLeader = this.user.isUserAdmin || this.user.roles.indexOf('leader') > -1;
```

Comment:
this is allowing admin or leader role only not leader of team based on code. But while testing I am seeing everyone being able to edit